velox/functions/prestosql/aggregates/tests/SumTest.cpp

```diff
@@ -163,6 +163,23 @@ TEST_F(SumTest, sumDouble) {
       vectors, {}, {"sum(c0)", "sum(c1)"}, "SELECT sum(c0), sum(c1) FROM tmp");
 }
 
+TEST_F(SumTest, sumDecimal) {
```

Comment:
Do we need to add more tests covering` NULL` cases and `GROUP BY`.

Reply:
There is a NULL value in the test. Do you have other NULL cases in mind?
I don't believe GROUP BY is required here as the PR scope is limited to Aggregate sum.
We should add GROUP BY tests separately.

Reply:
Yes, I see the NULLs. Thanks.